src/strategies/vested-deversifi/index.ts

```diff
@@ -0,0 +1,425 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'deversifi';
+export const version = '0.1.0';
+
+const vestedABI = [
+  {
+    inputs: [
+      { internalType: 'address', name: 'dvf', type: 'address' },
+      { internalType: 'address', name: 'xdvf', type: 'address' },
+      { internalType: 'address', name: 'vestingBeneficiary', type: 'address' },
+      { internalType: 'uint256', name: 'vestingStart', type: 'uint256' },
+      { internalType: 'uint256', name: 'cliffDuration', type: 'uint256' },
+      { internalType: 'uint256', name: 'vestingDuration', type: 'uint256' }
+    ],
+    stateMutability: 'nonpayable',
+    type: 'constructor'
+  },
+  {
+    anonymous: false,
+    inputs: [
+      {
+        indexed: false,
+        internalType: 'address',
+        name: 'token',
+        type: 'address'
+      },
+      {
+        indexed: false,
+        internalType: 'uint256',
+        name: 'amount',
+        type: 'uint256'
+      }
+    ],
+    name: 'TokensReleased',
+    type: 'event'
+  },
+  {
+    inputs: [],
+    name: '_delegateForVoting',
+    outputs: [{ internalType: 'address', name: '', type: 'address' }],
+    stateMutability: 'view',
+    type: 'function'
+  },
+  {
+    inputs: [],
+    name: '_dvf',
+    outputs: [{ internalType: 'contract IERC20', name: '', type: 'address' }],
+    stateMutability: 'view',
+    type: 'function'
+  },
+  {
+    inputs: [],
+    name: '_xdvf',
+    outputs: [
+      { internalType: 'contract IStakedDVF', name: '', type: 'address' }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  },
+  {
+    inputs: [],
+    name: 'beneficiary',
+    outputs: [{ internalType: 'address', name: '', type: 'address' }],
+    stateMutability: 'view',
+    type: 'function'
+  },
+  {
+    inputs: [],
+    name: 'cliff',
+    outputs: [{ internalType: 'uint256', name: '', type: 'uint256' }],
+    stateMutability: 'view',
+    type: 'function'
+  },
+  {
+    inputs: [],
+    name: 'delegateForVoting',
+    outputs: [{ internalType: 'address', name: '', type: 'address' }],
+    stateMutability: 'view',
+    type: 'function'
+  },
```

Comment:
You should actually read 'delegateForVoting' instead of 'beneficiary'. By default they are both the same, but its possible for someone to change the delgateForVoting to a different address than the beneficiary if they want.

Sorry I forgot to mention.